dashboards/azure-container-registry/azure-container-registry.json

```diff
@@ -71,8 +70,10 @@
             },
             "nrqlQueries": [
               {
-                "accountIds": [],
-                "query": "FROM Metric SELECT sum(azure.containerregistry.registries.SuccessfulPullCount) AS 'Total Successful Pull Count'  WHERE collector.name = 'azure-monitor' and azure.resourceType = 'microsoft.containerregistry/registries' TIMESERIES AUTO "
+                "accountIds": [
+                  11817994
```

Comment:
It looks like we have a hard-coded `accountId` here. Unless that is intentional, we should remove that.